articles/event-hubs/event-hubs-faq.yml

```diff
@@ -193,6 +193,12 @@ sections:
           How many partitions do I need?
         answer: |
           [!INCLUDE [event-hubs-partition-count](./includes/event-hubs-partition-count.md)]
+
+    questions:
```

Comment:
If you remove **questions:** line, the build should go through fine.

Reply:
Sure will do the changes

Reply:
Done the changes